Scellecs.Morpeh/Core/Components/ComponentId.cs

```diff
@@ -39,9 +39,11 @@ public static bool TryGet(int typeId, out Type type) {
     [Il2CppSetOption(Option.NullChecks, false)]
     [Il2CppSetOption(Option.ArrayBoundsChecks, false)]
     [Il2CppSetOption(Option.DivideByZeroChecks, false)]
-    internal static class ComponentId<T> where T : struct, IComponent {
+    public static class ComponentId<T> where T : struct, IComponent {
         internal static TypeInfo info;
         internal static bool initialized;
+
+        public static int StashSize;
```

Comment:
By default it will read 0 for the user, meaning that unless the user sets it, it's going to be an invalid value. I suppose we better add a constant here, so the user can do e.g. `ComponentId<T>.StashSize *= 2`